docs/tutorials/python/ndarray.md

```diff
@@ -63,7 +64,7 @@ and vice versa:
 
 ## Basic Element-wise Operations
 
-By default, `NDArray` performs element-wise operations:
+By default, a `NDArray` performs element-wise operations:
```

Comment:
"a" -> "an" because `NDArray` starts with a *vowel sound* (even though n is a consonant)